docs/nodes/statesync-node-quicksync.md

```diff
@@ -0,0 +1,246 @@
+
+```bash
+pkill commercionetworkd
+git init . 
+git remote add origin https://github.com/commercionetwork/commercionetwork.git
+git pull
+git checkout tags/$(cat .data | grep -oP 'Release\s+\K\S+')
+go mod verify
+make install
+```
+
+Test if you have the correct binaries version. Get version from chain data
+
+```bash
+cat .data | grep -oP 'Release\s+\K\S+'
+```
+
+Compare the result with the output of command
+
+```bash
+commercionetworkd version
+```
+
+Setup the validator node name. We will use the same name for node as well as the wallet key:
+
+```bash
+export CHAINID=commercio-$(cat .data | grep -oP 'Name\s+\K\S+')
+cat <<EOF >> ~/.profile
+export CHAINID="$CHAINID"
+EOF
+```
+
+Init the `.commercionetwork` folder with the basic configuration
+
+```bash
+commercionetworkd unsafe-reset-all --home ~/.commercionetwork
+# If you get a error because .commercionetwork folder is not present don't worry 
+
+commercionetworkd init $NODENAME --home ~/.commercionetwork
+# If you get a error because .commercionetwork folder is present don't worry 
+```
+
+Change the persistent peers inside `config.toml` file
+
+```bash
+sed -e "s|persistent_peers = \".*\"|persistent_peers = \"$(cat .data | grep -oP 'Persistent peers\s+\K\S+')\"|g" ~/.commercionetwork/config/config.toml > ~/.commercionetwork/config/config.toml.tmp
+mv ~/.commercionetwork/config/config.toml.tmp  ~/.commercionetwork/config/config.toml
+```
+
+Change the seeds inside the `config.toml` file
+```bash
+sed -e "s|seeds = \".*\"|seeds = \"$(cat .data | grep -oP 'Seeds\s+\K\S+')\"|g" ~/.commercionetwork/config/config.toml > ~/.commercionetwork/config/config.toml.tmp
+mv ~/.commercionetwork/config/config.toml.tmp  ~/.commercionetwork/config/config.toml
+```
+
+Change `external_address` value to contact your node using public ip of your node:
+```bash
+PUB_IP=`curl -s -4 icanhazip.com`
+sed -e "s|external_address = \".*\"|external_address = \"$PUB_IP:26656\"|g" ~/.commercionetwork/config/config.toml > ~/.commercionetwork/config/config.toml.tmp
+mv ~/.commercionetwork/config/config.toml.tmp  ~/.commercionetwork/config/config.toml
+```
+
+
+:::warning  
+At this point there may be some differences if you are using `KMS` with `HSM`.
+:::
+
+If you are using a kms you have to configure the node so that it can accept the connection from the sign provider.         
+Usually the connection is within a private lan network, or via vpn. Assuming that the validator node has an internal ip such as 10.1.1.1 the configuration to set up will be
+
+
+```toml
+priv_validator_laddr = "tcp://10.1.1.1:26658"
+```
+
+Edit `~/.commercionetwork/config/config.toml`, search `priv_validator_laddr`, and modify the configuration.
+
+**WARN**: the configuration of KMS must already be done. 
+
+
+## 4. Download the statesynced node snapshot
+
+Download the statesynced node snapshot and extract the data folder
+```
+wget "https://quicksync.commercio.network/$(echo $CHAINID)-statesync.latest.tgz" -P ~/.commercionetwork/
```

Comment:
I don't understand why you used `$(echo $CHAINID)` instead `$CHAINID`

Reply:
Fixed